squarewrapperapp/app/src/main/java/gmu/rqr/square_wrapper_app/CheckoutActivity.java

```diff
@@ -19,23 +51,200 @@ protected void onCreate(Bundle savedInstanceState){
         super.onCreate(savedInstanceState);
         //Uses checkout_layout.xml for layout and widgets
         setContentView(R.layout.checkout_layout);
+        initListItemLongClick();
         //Gets ArrayList of cart items that were passed as Extras from OrderActivity. CheckoutProduct must implement Serializable in order for this to work.
         checkoutProducts = (ArrayList<CheckoutProduct>) getIntent().getSerializableExtra("checkoutproducts");
     }
 
+    private void onCheckoutResult(Result<CheckoutResult, ResultError<CheckoutErrorCode>> result) {
+        if (result.isSuccess()) {
+            CheckoutResult checkoutResult = result.getSuccessValue();
+            String totalAmount = checkoutResult.getTotalMoney().format();
+            showDialog(getString(R.string.checkout_success_dialog_title, totalAmount),
+                    getString(R.string.checkout_success_dialog_message),
+                    new DialogInterface.OnClickListener() {
+                        public void onClick(DialogInterface dialog, int id) {
+                            // TODO: store successfully purchased items in transaction DB for reporting
```

Comment:
@natekob @jluckanavage This is where we have confirmation of a successful transaction and you should add the code to store the purchased items in the database for reporting